os/board/rtl8730e/src/component/bluetooth/profile/client/rtk_gcs_client.c

```diff
@@ -287,15 +287,14 @@ void general_client_notify_hdl(void *data)
 	gattc_dump(ntf_ind->len, ntf_ind->value, (uint8_t *)"notify event");
 	trble_data read_result;
 	read_result.length = ntf_ind->len;
-	read_result.data = malloc(read_result.length);
+	read_result.data = (uint8_t *)malloc(read_result.length);
```

Comment:
Here is no checking for malloc. As you already make a change for this kind of defect, I think you should check this as well.

Reply:
I have added checks for malloced pointers